modules/structs/activity.go

```diff
@@ -6,8 +6,11 @@ package structs
 import "time"
 
 type Activity struct {
-	ID        int64       `json:"id"`
-	UserID    int64       `json:"user_id"` // Receiver user
+	ID     int64 `json:"id"`
+	UserID int64 `json:"user_id"` // Receiver user
+	// the type of action
+	//
+	// enum: create_repo,rename_repo,star_repo,watch_repo,commit_repo,create_issue,create_pull_request,transfer_repo,push_tag,comment_issue,merge_pull_request,close_issue,reopen_issue,close_pull_request,reopen_pull_request,delete_tag,delete_branch,mirror_sync_push,mirror_sync_create,mirror_sync_delete,approve_pull_request,reject_pull_request,comment_pull,publish_release,pull_review_dismissed,pull_request_ready_for_review,auto_merge_pull_request
```

Comment:
This does add some duplication and takes a lot of horizontal space, but haven't found better ways of handling both of these concerns yet. Open to other ideas

Reply:
Normally inside a JSON schema, one would use [`$ref`](https://json-schema.org/understanding-json-schema/structuring#dollarref) to deduplicate, but I'm almost sure `go-swagger` does not support ref definition in source annotations, at least I could not find anything regarding it on http://goswagger.io/go-swagger/.